replica/database.cc

```diff
@@ -1051,6 +1051,20 @@ future<> database::add_column_family_and_make_directory(schema_ptr schema) {
     co_await cf.init_storage();
 }
 
+future<> database::for_each_table_gently(std::function<future<>(table_id, lw_shared_ptr<column_family>)> f) {
+    auto holder = co_await _cf_lock.hold_read_lock();
+    for (auto& cf: _column_families) {
```

Comment:
please use structured bindings so refs to "first" and "second" will have more meaningful names